test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -818,6 +799,8 @@ def __capture_trace_from_core(self, core_file_name, exec_path, out_path):
         gdb.execute("quit")
         quit()
         """ % (core_file_name)
+        # Remove tabs from triple quoted strings
+        gdb_python = gdb_python.replace("\t", "")
```

Comment:
Usually what I do in times like this is to use multiple double quoted strings which python merges back together again.  I see that Hiren made the comment to use the """, so he'll know best.  I'm fine with it.